plugins/catalog-react/src/components/EntityRefLink/EntityRefLink.tsx

```diff
@@ -46,7 +49,14 @@ export type EntityRefLinkProps = {
  */
 export const EntityRefLink = forwardRef<any, EntityRefLinkProps>(
   (props, ref) => {
-    const { entityRef, defaultKind, title, children, ...linkProps } = props;
+    const {
+      entityRef,
+      usePeekAheadPopover = false,
```

Comment:
I don't think this is necessary. It's either explicitly true or false or undefined which are both falsy.